src/test/java/apicurito/tests/configuration/TestConfiguration.java

```diff
@@ -43,6 +43,7 @@
     public static final String APICURITO_OPERATOR_IMAGE_URL = "apicurito.config.operator.image.url";
     public static final String APICURITO_UI_IMAGE_URL = "apicurito.config.ui.image.url";
     public static final String APICURITO_OPERATOR_METADATA_URL = "apicurito.config.operator.metadata.url";
+    public static final String APICURITO_OPERATOR_PREVIOUS_METADATA_URL = "apicurito.config.operator-prev-version.metadata.url";
```

Comment:
Add this param to `ReleaseSpecificParameters` since it will be not changing during testing one version of Apicurito.